CMakeLists.txt

```diff
@@ -52,10 +62,21 @@ if(NOT USE_SYSTEM_CURL OR NOT CURL_FOUND)
     set(BUILD_CURL_EXE OFF CACHE INTERNAL "" FORCE)
     set(BUILD_TESTING OFF CACHE INTERNAL "" FORCE)
     set(HTTP_ONLY ON CACHE INTERNAL "" FORCE)
-    if(NOT CMAKE_USE_OPENSSL)
+
+    if (USE_WINSSL OR USE_OPENSSL)
+        set(SSL_ENABLED ON CACHE INTERNAL "" FORCE)
+    else()
+        set(CURL_CA_PATH "none" CACHE INTERNAL "" FORCE)
```

Comment:
This is not required. Since CURL should use it's default paths for the CA files.

Reply:
This is required, cause in that case ssl is deactivated, and curl will complain while building